.github/vale/Vocab/Backstage/accept.txt

```diff
@@ -219,6 +219,10 @@ onboarding
 Onboarding
 OpenShift
 orgs
+padding
+Padding
+paddings
+Paddings
```

Comment:
It's enough to add just the lowercase version. If it's all lowercase, it'll allow the leading uppercase character anyway at start of sentence and similar.

Reply:
Will remove then.